aws/resource_aws_msk_cluster.go

```diff
@@ -699,6 +718,22 @@ func expandMskClusterTls(l []interface{}) *kafka.Tls {
 	return tls
 }
 
+func expandMskClusterScram(l []interface{}) *kafka.Sasl {
+	if len(l) == 0 || l[0] == nil {
+		return nil
+	}
+
+	m := l[0].(map[string]interface{})
```

Comment:
Nit: to err on the side of caution we can safely check this casting is correct e.g
```suggestion
	tfMap, ok := l[0].(map[string]interface{})
	if !ok {
	    return nil
	}
```